### ISSUE TITLE
Exception while launching Sublime

### ISSUE THREAD

**Issue by ehuss** [DamnWidget/anaconda#710]
## Expected Behaviour
Should start cleanly.

## Actual Behaviour
I get the following exception while launching Sublime:

```
Traceback (most recent call last):
  File "C:\Program Files\Sublime Text 3\sublime_plugin.py", line 389, in run_callback
    expr()
  File "C:\Program Files\Sublime Text 3\sublime_plugin.py", line 428, in <lambda>
    run_callback('on_load', callback, lambda: callback.on_load(v))
  File "C:\Users\Eric\AppData\Roaming\Sublime Text 3\Packages\Anaconda\listeners\linting.py", line 82, in on_load
    self.run_linter(view)
  File "C:\Users\Eric\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\linting\sublime.py", line 380, in run_linter
    Worker().execute(Callback(on_success=parse_results), **data)
  File "C:\Users\Eric\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\workers\market.py", line 86, in execute
    worker = cls.hire(cls)
  File "C:\Users\Eric\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\workers\market.py", line 31, in hire
    raw_interpreter = get_interpreter(active_view())
  File "C:\Users\Eric\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\helpers.py", line 416, in get_interpreter
    return get_settings(view, 'python_interpreter', 'python')
  File "C:\Users\Eric\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\helpers.py", line 254, in get_settings
    r = expand(view, r)
  File "C:\Users\Eric\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\helpers.py", line 273, in expand
    return tmp
UnboundLocalError: local variable 'tmp' referenced before assignment
```

## Steps to Reproduce
Launching Sublime with a bunch of windows and files open.

I can spend more time narrowing down the issue, but I think it's pretty clear that the `expand` function needs to handle when `window` is `None`.

## ST3, Anaconda and OS versions
Sublime Build 3144
Anaconda 2.1.23
Windows 10

**Comment by DamnWidget** [comment]
Hi, I noticed that myself yesterday and already fixed it but is not yet released, anyway, it doesn't really affects the plugin normal operations (apart from the annoying message in the console).

Is curious that at some point there is no Window object but obviously the Window entity exists (at least in the C++ scope).

**Comment by ehuss** [comment]
Yea, I've noticed there are race conditions in the Sublime API where `view.window()` returns None in the `on_load` event handler.  It's not always during startup, it can happen randomly.   I've resorted in some cases to using `set_timeout` as a hackish workaround.  Unfortunately even `is_loading` doesn't help and I can't reliably reproduce it (I just tried, and it took clicking around about a hundred files to get it to trigger).
